.eslintrc.js

```diff
@@ -74,6 +74,7 @@ const typescriptEslintRules = {
     '@typescript-eslint/no-unsafe-call': 'off', // OFF b/c used in Svelte components for UI logic
     '@typescript-eslint/no-unsafe-member-access': 'off', // OFF b/c there are simply too many linting errors
     '@typescript-eslint/no-unsafe-return': 'off', // OFF b/c used in Svelte components for UI logic
+    '@typescript-eslint/no-unsafe-argument': 'off',
```

Comment:
Would you mind adding a comment after explaining why it's `'off'` (for the sake of consistency)? :pray: